### PR TITLE
Make `download_file` require fewer permissions

### PR DESCRIPTION
**Pull request by Tombana**
From internal PR: https://github.com/plumerai/benchmark-worker-infra/pull/16/files#r945639351

> `get_bucket` makes an HTTP request which requires `storage.buckets.get` permissions. These are only part of the "Storage Object Admin" role which also grants write permission :warning: . The normal "Storage Object Viewer" for read only access does not have `storage.buckets.get`. Switching from `get_bucket` to `bucket` as recommended in the docs fixes this as it doesn't do a HTTP request.

I've tested the function locally.

### CODE REVIEW

File: practipy/gcloud.py [hunk 1]
```diff
@@ -112,7 +112,7 @@ def download_files(
     Note: paths are relative to `gs://<bucket_name>`!.
     """
 
-    bucket = gcs.Client(project=project).get_bucket(bucket)
+    bucket = gcs.Client(project=project).bucket(bucket_name)
```
Comment (Tombana) [inline practipy/gcloud.py:115]: The `bucket_name` is an unrelated change because my IDE complained about assigning a bucket to a `str`.

**Review by jneeven — APPROVED** [review]
Nice, thanks!
